.formatter.exs

```diff
@@ -1,4 +1,11 @@
 # Used by "mix format"
 [
-  inputs: ["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"]
+  locals_without_parens: [
+    plug: 1,
+    plug: 2
```

Comment:
Oh what does this mean?

Reply:
It wont wrap the arguments when we call `plug/1` or `plug/2` function 👍 
https://github.com/flipay/chopperbot/pull/12/files#diff-62c739eca8184d112d9c5d33d30e44feR8-R10